ksql-engine/src/test/java/io/confluent/ksql/integration/JoinIntTest.java

```diff
@@ -76,7 +76,7 @@ public void shouldLeftJoinOrderAndItems() throws Exception {
 
     final String queryString = String.format(
             "CREATE STREAM %s AS SELECT ORDERID, ITEMID, ORDERUNITS, DESCRIPTION FROM orders LEFT JOIN items " +
-                    " on orders.ITEMID = item.ITEMID WHERE orders.ITEMID = 'ITEM_1' ;",
+                    " on orders.ITEMID = items.ID WHERE orders.ITEMID = 'ITEM_1' ;",
```

Comment:
This was an existing bug which now fails due to the newly raised `KSQLException`?

Reply:
Yes, I fixed the test.